src/utils/response.js

```diff
@@ -0,0 +1,54 @@
+const codes = {
+  200: 'sucess',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
  200: "sucess",
```